### PR TITLE
added restaurant

### PR DESCRIPTION
**Pull request by igalvan**

### CODE REVIEW

**Comment by smanthena** [comment]
Any things to do?

File: Igalvan - Livingston [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Livingston
 Restaurants
+Mayrita's 
 Things to do
```
Comment (n0rbs) [inline Igalvan - Livingston:4]: No idea where this is @_@ 😕 Where is this and what do I do there for fun?

Comment (n0rbs) [inline Igalvan - Livingston:4]: @igalvan Try this out. You can drag an image to this form and it gets uploaded to github 😃 
screen shot 2016-06-22 at 2 15 59 am

Comment (igalvan) [inline Igalvan - Livingston:4]: Not much to do but enjoy nature. It's an agricultural area, lots of orchards: almond, peach, walnut, grapes... some lakes and rivers. Social clubs... hiphop, a big Hindu and Mexican community

**Pull request merged**